src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(health = 100, name) {
```

Comment:
you don't need to pass `health` here, every Animal should have health of `100` at creation